<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtDNI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DNI :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombres">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombres : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidoMaterno">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Materno</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidoPaterno">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Paterno</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDireccion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dirección :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTelefono">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Teléfono :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>771</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>2</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Registro de empleados :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnElimar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuitar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar</string>
    </property>
   </widget>
   <widget class="QTableView" name="tblEmpleados">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>511</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
